apps/protailwind/src/templates/exercise-template.tsx

```diff
@@ -175,10 +180,15 @@ const Video: React.FC<VideoProps> = React.forwardRef(
 )
 
 const GitHubLink: React.FC<{
-  exercise: Exercise
+  exercise: LessonResource
   module: SanityDocument
 }> = ({exercise, module}) => {
-  const github = exercise.github ?? exercise.solution?.github
+  const router = useRouter()
+  const {data: resources} = trpc.resources.byExerciseSlug.useQuery({
+    slug: router.query.exercise as string,
+    type: exercise._type,
+  })
```

Comment:
this could be `/api/skill/resources/lesson?lessonSlug=XXXX&type=XXXXX` where in the `[skill]` api config we pass in a `lessonLoader` callback that would give us local control of what resources were loaded.